http-server/src/main/java/de/unistuttgart/iaas/amyassist/amy/httpserver/cors/CORSFilter.java

```diff
@@ -92,10 +106,25 @@ private void preflight(ContainerRequestContext requestContext, String origin) {
 	}
 
 	private void checkOrigin(ContainerRequestContext requestContext, String origin) {
-		if (!origin.equals(ALLOWED_ORIGIN)) {
+		if (!getAllowedOrigins().contains(origin) && !origin.matches(LOCALHOST_REGEX)) {
 			requestContext.setProperty(ACCESS_DENIED, Boolean.TRUE);
 			throw new WebApplicationException(origin + " not allowed", Status.FORBIDDEN);
 		}
 
 	}
+
+	/**
+	 * gets all allowed origins
+	 * 
+	 * @return a list of allowed origins
+	 */
+	public List<String> getAllowedOrigins() {
+		if (config == null) {
+			config = configurationLoader.load(CONFIG_NAME);
+		}
+		// Load pipe-separated list of allowed origins from config
+		String[] origins = config.getProperty(CONFIG_ORIGINS_KEY).split("\\|");
```

Comment:
use a comma separated list, instead of `|`

Reply:
@Legion2 URLs may contain commas, that‘s why I chose the pipe symbol

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin comma is not allowed in a origin `<scheme> "://" <hostname> [ ":" <port> ]` and commas are used to separate multiple origins in the header. So we can use comma to separate allowed origins in the config

Reply:
the list just needs a seperator, there is no gain from changing to a comma